test/k8sT/Chaos.go

```diff
@@ -77,15 +77,26 @@ var _ = Describe("K8sChaosTest", func() {
 
 		for _, pod := range pods {
 			for _, ip := range dsPods {
+				By("\tPinging test-ds service pod with IP %s from client pod %s", ip, pod)
 				res := kubectl.ExecPodCmd(
 					helpers.DefaultNamespace, pod, helpers.Ping(ip))
 				log.Debugf("Pod %s ping %v", pod, ip)
 				ExpectWithOffset(1, res).To(helpers.CMDSuccess(),
 					"Cannot ping from %q to %q", pod, ip)
 
-				err = kubectl.WaitForKubeDNSEntry(testDSService)
+				By("\tWaiting for kube-dns entry for service testds-service")
+				err = kubectl.WaitForKubeDNSEntry(testDSService, helpers.DefaultNamespace)
 				ExpectWithOffset(1, err).To(BeNil(), "DNS entry is not ready after timeout")
 
+				By("\tGetting ClusterIP For testds-service")
+				host, _, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, "testds-service")
```

Comment:
That can be out of the loop too, is a global thing for all pods.

Reply:
Will be fixed in next upload.